<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserFormClass</class>
 <widget class="QMainWindow" name="UserFormClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>655</width>
    <height>508</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UserForm</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QListWidget" name="listWidgetQuestions">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>331</width>
      <height>401</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidgetAnswers">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>10</y>
      <width>251</width>
      <height>401</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditQ">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>430</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditA">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>470</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonQ">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>430</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Question</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonA">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>470</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Answer</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="UserForm.qrc"/>
 </resources>
 <connections/>
</ui>
